src/composables/useFlawModel.ts

```diff
@@ -44,18 +39,21 @@ export function useFlawModel(forFlaw: ZodFlawType = blankFlaw(), emit: FlawEmitt
   const osimLink = computed(() => getFlawOsimLink(flaw.value.uuid));
 
   async function createFlaw() {
-    postFlaw(flaw.value)
+    isSaving.value = true;
```

Comment:
Isn't `isSaving` being handled twice for update/create actions, as it is still on `FlawForm` `onSubmit`?

Reply:
Yes, thank you!